site/src/sphinx/_extensions/api.py

```diff
@@ -0,0 +1,96 @@
+from docutils.parsers.rst.roles import register_canonical_role, set_classes
+from docutils.parsers.rst import directives
+from docutils import nodes
+from sphinx.writers.html import HTMLTranslator
+from sphinx.errors import ExtensionError
+
+import os
+import re
+
+
```

Comment:
additional line?

Reply:
It's Python convention to use two empty lines between methods.